ivadomed/scripts/automate_training.py

```diff
@@ -364,18 +396,17 @@ def automate_training(config, param, fixed_split, all_combin, n_iterations=1, ru
         compute_statistics(results_df, n_iterations, run_test)
 
 
-def main():
-    init_ivadomed()
-
+def main(args=None):
+    imed_utils.init_ivadomed()
     parser = get_parser()
-    args = parser.parse_args()
+    args = imed_utils.get_arguments(parser, args)
 
     # Get thr increment if available
     thr_increment = args.thr_increment if args.thr_increment else None
 
-    # Run automate training
-    automate_training(args.config, args.params, bool(args.fixed_split), bool(args.all_combin), int(args.n_iterations),
-                      bool(args.run_test), args.all_logs, thr_increment, bool(args.multi_params))
+    automate_training(args.config, args.params, bool(args.fixed_split), bool(args.all_combin),
```

Comment:
Perhaps it would be cleaner if each argument was placed in its own line? see: https://github.com/ivadomed/ivadomed/wiki/code-style

Reply:
Good call!

Reply:
Fixed here: https://github.com/ivadomed/ivadomed/pull/580/commits/353dcaaf82ef7004812c968028b19c5d1cead2ae